libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx

```diff
@@ -125,6 +130,15 @@ const DocumentScreen: FC<Props> = ({ title, intro, listPath }) => {
               />
             </GridColumn>
           </GridRow>
+          <Box marginTop={3}>
+            <Input
+              label="Leit"
+              name="Search1"
+              placeholder="Sláðu inn leitarorð"
+              size="sm"
+              onChange={(e) => setQ(e.target.value)}
```

Comment:
This input is uncontrolled. My preference would be to pass it `value={q}` to avoid annoying surprises later on.

```suggestion
              onChange={(e) => setQ(e.target.value)}
              value={q}
```